<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgNucleo</class>
 <widget class="QWidget" name="WdgNucleo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>62</width>
    <height>96</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
   <item>
    <widget class="QLabel" name="lblId">
     <property name="text">
      <string>id</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wdgProcesso" native="true"/>
   </item>
   <item>
    <widget class="QLabel" name="lblStatus">
     <property name="text">
      <string>status</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
